RELEASING.md

```diff
@@ -0,0 +1,57 @@
+# Release Steps
+
+To release artifacts 'dependencies-parent', 'dependencies', and 'linkage-checker-enforcer-rules',
+follow the steps below.
+
+Note: Cloud OSS BOM release procedure [boms/cloud-oss-bom/RELEASING.md](boms/cloud-oss-bom/RELEASING.md)
+is not part of this pipeline.
+
+## 1. Create a tag for a release
+
+```
+git checkout origin/master
+git pull
+scripts/prepare_release.sh X.Y.Z    # X.Y.Z is version number (without prefix 'v')
+```
+
+This script creates a branch and a tag with the version number.
+
+### Reverting prepare_release.sh
+
+Sometimes subsequent builds fail and you want to start over with the same version number with
+updated master by reverting this step.
+To revert the effect of this step, delete the tag and the branch. For example, to revert version
+'0.1.1', do the following steps:
```

Comment:
You shouldn't need to delete the tag. Just create v1.1.2A, v1.1.2B, etc.

Reply:
That makes difficult to find corresponding commit in [releases page](https://github.com/GoogleCloudPlatform/cloud-opensource-java/releases) from Maven central version.